rs-matter/src/persist.rs

```diff
@@ -84,10 +103,10 @@ pub mod fileio {
         ) -> Result<(), Error> {
             let dir = dir.as_ref();
 
-            self.load(dir, matter)?;
+            //self.load(dir, matter)?;
```

Comment:
I don't follow why this is removed?

Reply:
Good catch. This is not trivial and it needs to be documented.

I've put a comment now just on top of the commented line which explains why it needs to stay commented.

Fwiw, this problem does not exist in downstream `rs-matter-stack` which uses a slightly different (and more opinionated) persistent metaphor. But whether to merge `rs-matter-stack` upstream is still future.

Reply:
Comment in this commit: https://github.com/project-chip/rs-matter/pull/232/commits/b720c55654688901d3dc98c21ee2773b587ae695